mitodl/open-discussions
# remove cost and availability facets

#### Pre-Flight checklist

#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/2918

#### What's this PR do?
This removes cost and availability facet filters from the search page.
#### How should this be manually tested?
Go to http://localhost:8063/learn/search. There should be no cost or availability filters. Text search and the remaining filters should continue to work as expected.  Cost and availability should still be displayed the same way as it was previously in learning resource cards. This will be updated in a future pr.

<img width="377" alt="Screen Shot 2020-05-13 at 11 43 09 AM" src="https://user-images.githubusercontent.com/1934992/81835042-c62b5c80-950f-11ea-8c0e-72fb5d4a2494.png">
<img width="1678" alt="Screen Shot 2020-05-13 at 11 42 31 AM" src="https://user-images.githubusercontent.com/1934992/81835045-c6c3f300-950f-11ea-8384-43d0ffd86881.png">
